README.md

```diff
@@ -118,35 +118,57 @@ override fun onCreate() {
 <br>
 
 ## Optional Parameters
-You can add optional parameters like userId anytime and update the instance of Sign3Intelligence.
+1.	You can add optional parameters like UserId, Phone Number, etc., at any time and update the instance of Sign3Intelligence.
+2. To update the Sign3Intelligence instance with optional parameters, including additional attributes, you can use the following examples.
+3. **NOTE:** You need to call **getIntelligence()** function whenever you update options.
```

Comment:
callout that once options are updated, it gets reset. Clients need to explicitly update the options again to ingest again or else default value of OTHERS will be sent to backend